src/parser/jobs/sam/modules/Kaiten.tsx

```diff
@@ -109,40 +112,43 @@ export class Kaiten extends Analyser {
 	}
 
 	private onComplete() {
-		this.suggestions.add(new TieredSuggestion({
-			icon: this.data.actions.HISSATSU_KAITEN.icon,
-			content: <Trans id = "sam.kaiten.suggestion.badkaiten.content">
-				Avoid using <DataLink action="HISSATSU_KAITEN"/> on any GCDs besides <DataLink action ="IAIJUTSU"/> moves and <DataLink action = "OGI_NAMIKIRI"/>. These actions are worth it because of the potency gained per kenki spent.
-			</Trans>,
-			tiers: SEVERITY_TIERS,
-			why: <Trans id ="sam.suggestion.badkaiten.why">
-					You used Kaiten  <Plural value={this.badKaitens} one="# time" other="# times"/> on non-optimal GCDs.
-			</Trans>,
-			value: this.badKaitens,
-		}))
-
-		this.suggestions.add(new TieredSuggestion({
-			icon: this.data.actions.HISSATSU_KAITEN.icon,
-			content: <Trans id = "sam.kaiten.suggestion.missedkaiten.content">
-				Always use <DataLink action = "HISSATSU_KAITEN"/> on <DataLink action = "MIDARE_SETSUGEKKA"/>,<DataLink action = "HIGANBANA"/> and <DataLink action="OGI_NAMIKIRI"/>. The gain on these actions from kaiten is too great to miss.
-			</Trans>,
-			tiers: SEVERITY_TIERS,
-			why: <Trans id ="sam.suggestion.missedkaiten.why">
-					You forgot to use Kaiten  <Plural value={this.missedKaitens} one="# time" other="# times"/> on optimal GCDs.
-			</Trans>,
-			value: this.missedKaitens,
-		}))
-
-		this.suggestions.add(new TieredSuggestion({
-			icon: this.data.actions.HISSATSU_KAITEN.icon,
-			content: <Trans id = "sam.kaiten.suggestion.doublekaiten.content">
-				Avoid using <DataLink action = "HISSATSU_KAITEN"/> when already under the effect of it.
-			</Trans>,
-			tiers: SEVERITY_TIERS,
-			why: <Trans id ="sam.suggestion.doublekaiten.why">
-					You used Kaiten  <Plural value={this.doubleKaitens} one="# time" other="# times"/> when you already had it up.
-			</Trans>,
-			value: this.doubleKaitens,
-		}))
+		// No suggestions if after patch 6.1
+		if (this.parser.patch.before('6.1')) {
```

Comment:
You initialise all these vars as 0 and they use TieredSuggestions, so they won't throw anyway. Here too tho, consider just returning early in the function if it's 6.1+ I guess.

Reply:
removed the redundancy.